server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -162,9 +163,16 @@ export function create(
 				? uuid()
 				: (request.body.id as string) || uuid();
 
+			Lumberjack.info(
+				`Whole summary on First Summary ${request.body.enableWholeSummaryTreeOnFirstSummary}.`,
+			);
 			// Summary information
-			const summary = request.body.summary;
+			const summary =
```

Comment:
This PR is great for demonstration purposes. But I believe old & new way of doing things is incompatible, and thus server changes needs to be checked in separately, and fully deploy to 100% prod before we can make client changes.

Reply:
Or we need it to be configurable (enable/disable) on the driver side so that we can have a follow-up PR to enable this in a specific driver version.